conf/composites.conf

```diff
@@ -203,6 +203,16 @@ composites {
     policy = "leave";
     description = "Message contains redirector, anonymous or IPFS gateway URL and is marked by fuzzy/bayes/SURBL/RBL";
   }
+  MIME_BAD_UNICODE_NNBSP {
+    expression = "^MIME_BAD_UNICODE[/^0x202fd after /i]";
+    description = "Exclude NNBSP (narrow no-break space) from MIME_BAD_UNICODE which used by MacOS Mail.app";
```

Comment:
```diff
-description = "Exclude NNBSP (narrow no-break space) from MIME_BAD_UNICODE which used by MacOS Mail.app";
+description = "Exclude NNBSP (narrow no-break space) from MIME_BAD_UNICODE which is used by MacOS Mail.app";
```

Reply:
@moisseev I also found that there other cases, in total it's quite big list:
```
200bd
200cd
200dd
200ed
200fd
2028d
2029d
202ad
202bd
202cd
202ed
202fd
2068d
2069d
206ad
206cd
206ed
206fd
feffd
```

All them are special chars if strip last "d", reference: https://www.compart.com/en/unicode/U+202e